indent/javascript.vim

```diff
@@ -458,7 +459,7 @@ function GetJavascriptIndent()
   if l:line =~ '^[])}]\|^|}'
     return max([indent(num),0])
   elseif num
-    return indent(num) + s:W + switch_offset + bL + isOp
+    return max([indent(num) + get(l:,'case_offset',s:W) + switch_offset + bL + isOp, 0])
```

Comment:
case_offset, if existing, replaces the &sw which is added for lines between `{}`. it isn't additional indent